### ISSUE TITLE
Research Question of Interest

### ISSUE THREAD

**Issue by JennyonOort** [UBC-MDS/diabetes_predictor_py#92]
Also another question to hear everyone's thoughts, what should be our research question, please vote on 1 or 2 and state your reasons: 

1. whether clinical features such as glucose levels, BMI, pregnancies, and etc. can effectively predict diabetes in Pima Indian women. 
2. Can logistic regression effectively predict diabetes in Pima Indian women using clinical features such as glucose levels, BMI, pregnancies, and etc.

**Comment by InderKhera** [comment]
I believe we should do the first one, the report should focus on the research itself and not the model. It can be the case that if the LogisticRegression did not classify well another may model may have been able to.

**Comment by kuo4230** [comment]
I vote for #2 as most of our results were evaluating the model itself (e.g. accuracy) using the listed features. We did not exactly carry out feature engineering, feature selection, or feature evaluation in the report. With question two, the way we did the research can be more leniently accepted as we were indeed using LR with limited data (features) to assess if it could be a reasonable model to predict diabetes; however, with question one, we might actually need to dig deeper and reference other research papers as well as carry out additional experiments and discussions to proof that we have carefully picked out these clinical features as the input.
